packages/quantic/force-app/main/default/lwc/quanticNoResults/quanticNoResults.js

```diff
@@ -0,0 +1,89 @@
+import { api, LightningElement, track } from 'lwc';
+import { initializeWithHeadless, registerComponentForInit } from 'c/quanticHeadlessLoader';
+
+export default class QuanticNoResults extends LightningElement {
+    /** @type {string}*/
+    @api engineId;
+
+    /** @type {import("coveo").SearchStatus} */
+    searchStatus;
+
+    /** @type {import("coveo").HistoryManager} */
+    historyManager;
+
+    /** @type {import("coveo").QuerySummary} */
+    querySummary;
+
+    /** @type {() => void} */
+    unsubscribeSearchStatus;
+    /** @type {() => void} */
+    unsubscribeHistoryManager;
+    /** @type {() => void} */
+    unsubscribeQuerySummary;
+    /**
+     * @type {boolean}
+     */
+    @api enableCancelLastAction
```

Comment:
Salesforce seems to have this rule that boolean props can't be initialized to `true`. I don't love this rule but if we have to follow it, I think it would be better to phrase this prop in the negative so that the desired default behaviour is still enforced without the user having to pass an attribute.

ex.
```
@api disabledCancelLastAction = false
```

so the user just needs to do
```
<c-quantic-no-results engine-id={engineId}><c-quantic-no-results>
```
to get the default behaviour.

What do you think?

Reply:
wonderful idea, I m taking it